src/Compilers/CSharp/Test/Syntax/Parsing/LocalFunctionParsingTests.cs

```diff
@@ -71,28 +71,17 @@ void M6()
                         N(SyntaxKind.Block);
                         {
                             N(SyntaxKind.OpenBraceToken);
-                            N(SyntaxKind.LocalDeclarationStatement);
-                            {
-                                N(SyntaxKind.VariableDeclaration);
-                                {
-                                    N(SyntaxKind.IdentifierName);
-                                    {
-                                        N(SyntaxKind.IdentifierToken, "await");
-                                    }
-                                    N(SyntaxKind.VariableDeclarator);
-                                    {
-                                        N(SyntaxKind.IdentifierToken, "L");
-                                    }
-                                }
-                                M(SyntaxKind.SemicolonToken);
-                            }
                             N(SyntaxKind.ExpressionStatement);
```

Comment:
this is a better parse than before.  you can't have local decls with `<` in their identifier name, so this being parsed as a misplaced expression makes much more sense.